test/ext/chainrulescore.jl

```diff
@@ -0,0 +1,40 @@
+using Enzyme
+using Test
+using ChainRules
+using ChainRulesCore
+using LinearAlgebra
+using EnzymeTestUtils
+
+# utility functions
+fdiff(f, x::Number) = autodiff(Forward, f, Duplicated, Duplicated(x, one(x)))[2]
+rdiff(f, x::Number) = autodiff(Reverse, f, Active, Active(x))[1][1]
+rdiff(f, x::AbstractArray) = autodiff(Reverse, f, Active, Duplicated(x, zero(x)))[1][1]
```

Comment:
This probably doesn't do what you want. derivatives of vectors are stored in place [aka in that zero(x) temporary]